.travis.yml

```diff
@@ -21,15 +21,19 @@ install:
   - yarn --pure-lockfile
 before_script:
   - "psql -c 'create database travis_ci_test;' -U postgres"
+  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+  - chmod +x ./cc-test-reporter
 script:
   - scripts/lint-travis.sh
   - yarn test
+  - if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT; fi
 env:
   global:
   - CF_API="https://api.fr.cloud.gov"
   - CF_ORGANIZATION="gsa-18f-federalist"
   - CXX=g++-4.8
   - YARN_VERSION=0.24.6-1
+  - CC_TEST_REPORTER_ID=101a439651b6abd27e5440028a53b5b8f08fe0889f3948d5f58f6cf9f7c786a0
```

Comment:
And actually I'm not even sure this will work. We might need to generate the coverage report(s) from our test runner so that it has something to submit back to CodeClimate.